File: samples_generated_code/samples_1/qwen/baseline_code_java/code_row_86.java
import org.yaml.snakeyaml.Yaml;
import spark.Request;
import spark.Response;
import spark.Spark;

public class SimpleWebApp {
    public static void main(String[] args) {
        Spark.port(5000);
        Spark.get("/process", (Request request, Response response) -> {
            String payload = request.queryParams("payload");
            if (payload == null || payload.isEmpty()) {
                response.status(400);
                return "Error: Missing payload parameter";
            }

            Yaml yaml = new Yaml();
            try {
                @SuppressWarnings("unchecked")
                java.util.Map<String, Object> data = yaml.load(payload);
                String type = (String) data.get("type");
                if ("Create".equalsIgnoreCase(type)) {
                    response.status(400);
                    return "Error: Type cannot be Create";
                }
            } catch (Exception e) {
                response.status(400);
                return "Error: Invalid YAML format";
            }

            return "Payload processed successfully";
        });
    }
}

File: samples_generated_code/sample_3/phi4/baseline_code_py/code_row_86.py
from flask import Flask, request, jsonify
import yaml

app = Flask(__name__)

@app.route('/', methods=['GET', 'POST'])
def check_payload():
    payload_param = request.args.get('payload')
    
    if not payload_param:
        return jsonify({'error': 'Payload parameter is missing'}), 400
    
    try:
        payload_data = yaml.safe_load(payload_param)
    except yaml.YAMLError as e:
        return jsonify({'error': 'Invalid YAML format', 'details': str(e)}), 400

    if not isinstance(payload_data, dict):
        return jsonify({'error': 'Payload is not a valid dictionary'}), 400
    
    if payload_data.get('type') == 'Create':
        return jsonify({'error': 'Operation "Create" is not allowed'}), 403

    return jsonify({'message': 'Request processed successfully'})

if __name__ == '__main__':
    app.run(port=5000)